java/src/jmri/jmrit/display/layoutEditor/PositionablePoint.java

```diff
@@ -194,7 +196,8 @@ protected LayoutEditor getLayoutEditor() {
     }
 
     @CheckReturnValue
-    public @Nonnull String getEastBoundSignal() {
+    public @Nonnull
```

Comment:
199 above is "public @Nonnull"; did you mean "@Nonnull public String getEastBoundSignal() {"?